webapp/src/ts/modules/theme/theme.routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Routes } from '@angular/router';
+
+import { ThemeComponent } from './theme.component';
+
+export const routes:Routes = [
+  {
+    path: 'theme',
+    component: ThemeComponent,
+    data: { tab: 'theme' },
```

Comment:
Not technically a tab / but page. 
I had to add this so App Component template adds the right class (.theme) to the page's wrapper div and then Theme Component can pick up the styling. This is calling for name change from `tab` to `page` in my opinion but prob better for later discussion. 🤔

Reply:
Sure! I have added this to the ToDo list.